include/aspect/postprocess/composition_BPL_statistics.h

```diff
@@ -0,0 +1,54 @@
+/*
```

Comment:
Please unify the name of the header file and the source file. I would prefer composition_limiter_statistics over abbreviations like BPL, since most people (including me) would not know what it means had they not talked to you before.